packages/socketio/package.json

```diff
@@ -53,20 +53,20 @@
     "@feathersjs/commons": "^5.0.0-pre.15",
     "@feathersjs/feathers": "^5.0.0-pre.15",
     "@feathersjs/transport-commons": "^5.0.0-pre.15",
-    "socket.io": "^4.3.1"
+    "socket.io": "^4.4.0"
   },
   "devDependencies": {
     "@feathersjs/commons": "^5.0.0-pre.2",
```

Comment:
Still got a potential issue here line 59.
Supposed to be removed as it is already in dependencies?

Reply:
Ah, I was confused before because I thought it was about the `socket.io` dependency change.

Reply:
Good catch, fixed now. Thanks!

Reply:
No worries, the line comments do seem to show a bit weird sometimes haha 👍🏻